src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas3/TransitionalAccommodationReferralReportsTest.kt

```diff
@@ -184,6 +184,7 @@ class TransitionalAccommodationReferralReportsTest : IntegrationTestBase() {
             )
           }
           withPrisonNameAtReferral("HM Hounslow")
+          withPersonReleaseDate(LocalDate.now())
```

Comment:
we don't appear to be asserting on this value anywhere in the test? (unless i'm missing it)

Reply:
Good one, silly me..i have added assertion now.